config/ant-phase-verify.xml

```diff
@@ -39,9 +39,78 @@
     </checkstyle>
 
     <tstamp>
-      <format property="FINISHED" pattern="dd/MM/yyyy hh:mm:ss aa" />
+      <format property="FINISHED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
     </tstamp>
-    <echo>Checkstyle finished: ${FINISHED}</echo>
+    <echo>Checkstyle finished (checkstyle_checks.xml) : ${FINISHED}</echo>
+
+    <tstamp>
+      <format property="STARTED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
+    </tstamp>
+    <echo>Checkstyle started (checkstyle_non_main_files_checks.xml): ${STARTED}</echo>
+
+    <checkstyle config="config/checkstyle_non_main_files_checks.xml"
+                failOnViolation="true"
+                failureProperty="checkstyle.failure.property">
+      <path>
+        <fileset dir="${basedir}" includes="**/*">
+          <exclude name="src/it/resources/**/*" />
+          <exclude name="src/test/resources/**/*" />
+          <exclude name="src/test/resources-noncompilable/**/*"/>
+
+          <exclude name=".git/**/*"/>
+          <exclude name="src/main/**/*"/>
+          <exclude name="src/test/java/**/*"/>
+          <exclude name="src/site/resources/images/**/*"/>
+
+          <!-- Eclipse project files -->
+          <exclude name=".settings/**/*"/>
+          <exclude name=".externalToolBuilders/**/*"/>
+          <exclude name=".classpath"/>
+          <exclude name=".project"/>
+
+          <!-- m2e-code-quality Eclipse IDE plugin temporary configuration files for Eclipse CS Checkstyle / PMD / FindBugs Plug-Ins -->
+          <exclude name=".checkstyle/**/*"/>
+          <exclude name=".pmd/**/*"/>
+          <exclude name=".pmdruleset.xml"/>
+          <exclude name=".fbExcludeFilterFile/**/*"/>
+
+          <!-- NetBeans project files -->
+          <exclude name="**/nbactions.xml"/>
+          <exclude name="**/nb-configuration.xml"/>
+
+          <!-- Maven build folder -->
+          <exclude name="target/**/*" />
+          <exclude name="bin/**/*" />
+
+          <!-- IDEA project files -->
+          <exclude name="*.iml"/>
+          <exclude name="*.ipr"/>
+          <exclude name="*.iws"/>
+          <exclude name=".idea/**/*"/>
+
+          <!-- Temp files -->
+          <exclude name="*~" />
+
+          <!-- Java Virtual machine crash logs -->
+          <exclude name="hs_err_pid*"/>
+          <exclude name="replay_pid*"/>
+
+          <!-- Apple MAC OSX hidden file -->
+          <exclude name=".DS_Store" />
+
+          <!-- NonDex files -->
+          <exclude name=".nondex/**/*" />
+        </fileset>
+      </path>
+      <formatter type="plain"/>
+      <formatter type="xml" toFile="${mvn.project.build.directory}/cs_errors.xml"/>
```

Comment:
The other ant task has the same filename for `toFiles`. Shouldn't they be unique to prevent clashing?
We should name this `cs_non_main_files_errors.xml`.